public/scripts/main.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(() => {
+    function onKonamiCode(cb) {
+        let input = '';
+        const key = '38384040373937396665';
+
+        document.addEventListener('keydown', e => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
        document.addEventListener('keydown', (e) => {
```